WcaOnRails/app/assets/javascripts/currencies.js.erb

```diff
@@ -0,0 +1,60 @@
+window.wca = window.wca || {};
+<%
+  # Select currencies, for all the iso_code is matching the hash key.
+  # However for 2 currencies (JPY, GHS) there is a second currency with the same iso_code but a different key.
+  # They exist for backward compatibilities reason (see here: https://github.com/RubyMoney/money/blob/190683e7b84b66d11b83b7be32d019cf3c2fc114/config/currency_backwards_compatible.json), and we actually don't use them.
+  selected_currencies = Money::Currency.table.select { |k,v| k.to_s == v[:iso_code].downcase }
+  currencies_info_hash = Hash[selected_currencies.map do |k,v|
+    [v[:iso_code], {:symbol => v[:symbol],
+                    :symbol_first => v[:symbol_first],
+                    :subunit_to_unit => v[:subunit_to_unit]}]
+  end]
+%>
+wca._currenciesInfo =
+<%= raw(JSON.pretty_generate(currencies_info_hash, indent: '', object_nl: '', array_nl: '', space: '', space_after: '')) %>;
+
+wca.getCurrencyInfo = function(isoCode) {
+  return (isoCode in wca._currenciesInfo) ? wca._currenciesInfo[isoCode] : wca._currenciesInfo['USD'];
```

Comment:
I think a more javascripty way of writing this would be:

```javascript
  return wca._currenciesInfo[isoCode] || wca._currenciesInfo['USD'];

```